resources/assets/coffee/_classes/twitch-player.coffee

```diff
@@ -9,7 +9,9 @@ class @TwitchPlayer
 
 
   initializeEmbed: =>
-    @turbolinksReload.load 'https://player.twitch.tv/js/embed/v1.js', @startAll
+    @turbolinksReload
+      .load 'https://player.twitch.tv/js/embed/v1.js'
+      .then @startAll
```

Comment:
`load` can be `undefined` by definition even though checking if `Twitch` is set should prevent it. Coming to think of it, wouldn't this loop infinitely if `Twitch` isn't set by the script? 🤔

Reply:
the `load` wouldn't do anything if the script is already loaded